### ISSUE TITLE
Some doubts about the code details

### ISSUE THREAD

**Issue by WEIZHIHONG720** [MCG-NJU/VideoMAE#56]
First of all, thank you for your excellent work. I have some questions to ask.

**In the run_videomae_vis.py** 
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/43931281/191404592-a75bcf12-4ee2-4304-9374-aa09d74cd84a.png">

Looking forward to your reply, thank you!

**Comment by yztongzhan** [comment]
Hi @WEIZHIHONG720! Thanks for your question!
1. The target of pre-training phase is the normalized cube pixels.
2. We follow the implementation in [MAE_pytorch](https://github.com/pengzhiliang/MAE-pytorch/blob/8b3ab30f99000ac56b2469de6fd48341ed799a21/run_mae_vis.py#L127).
